src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -75,6 +75,24 @@ def function_converter(self) -> Mapping[str, fields.MetricsFunction]:
                     ),
                     default_result_type="integer",
                 ),
+                fields.MetricsFunction(
+                    "count",
+                    snql_distribution=lambda args, alias: Function(
+                        "countIf",
+                        [
+                            Column("value"),
+                            Function(
+                                "equals",
+                                [
+                                    Column("metric_id"),
+                                    self.resolve_metric("span.duration"),
```

Comment:
Same as transaction.duration, we collect span.duration for each span, so we can just count the number of rows that have span.duration to get span count